### PR TITLE
fix and add crafting recipes

### PR DESCRIPTION
**Pull request by Niklp09**
#9

### CODE REVIEW

**Comment by S-S-X** [comment]
```
Checking home_vending_machines/crafts.lua         3 warnings
    home_vending_machines/crafts.lua:22:8: accessing undefined variable farming
    home_vending_machines/crafts.lua:22:20: accessing undefined variable farming
    home_vending_machines/crafts.lua:22:36: accessing undefined variable farming
```

To fix checks just add "farming" to read_globals table here: https://github.com/mt-mods/home_workshop_modpack/blob/eacbe80faec8be21e774bd2f9accad4ac33d3cfd/.luacheckrc#L23-L24

File: home_vending_machines/crafts.lua [hunk 1]
```diff
@@ -19,7 +19,7 @@ if minetest.get_modpath("default") then
             },
         })
     end
-    if minetest.get_modpath("farming") then
+    if farming and farming.mod and farming.mod == "redo" then
```
Comment (S-S-X) [inline home_vending_machines/crafts.lua]:
```suggestion
    if farming and farming.mod == "redo" then
```
Truth value check is unnecessary and also easier to read without it.

**Review by wsor4035 — APPROVED** [review]
note: my approval is pending ssx's two changes being commited/fixed

**Review by BuckarooBanzay — APPROVED** [review]

**Comment by BuckarooBanzay** [comment]
> note: my approval is pending ssx's two changes being commited/fixed

Same, the rest looks good :+1:

**Pull request merged**